registry/donor/views.py

```diff
@@ -106,6 +106,21 @@ def detail(rc):
     )
 
 
+@blueprint.get("/award_document/<medal_id>/<rc>")
```

Comment:
Just an idea - the URL - maybe changing it to something else - `"/award_document/<rc>/<medal_id>"`? This is probably just a matter of taste, or a point for discussion.

By the way, by changing the URL manually, it is possible to print a document for a medal, that hasn't jet been awarded.

Third idea regarding the URL - the medal ID is just a number. Would it be useful to change it to the medal "name", eg. _medal_br_

Reply:
Printing a document prior to awarding a medal might be useful in some cases - printing them before the ceremony, before sending them out. Depends on how much we want to limit the end users.

Reply:
:bulb: Use the medal slug instead of the medal ID?

Reply:
.

Reply:
> 💡 Use the medal slug instead of the medal ID?

We will need to replace 
```py
Medal.query.get(medail_id)
``` 
with 
```py 
Medal.query.filter_by(medal_slug=medal_slug).first()
``` 
or something like that